docs/development.md

```diff
@@ -64,9 +60,19 @@
 
 ## Development notes
 
+- Set version numbers:
+    ```sh
+    $ export PRESIDIO_LABEL=[version number]
```

Comment:
by saying `version number` do you mean `label`? I mean in docker terminology

Reply:
should be label, you're right